gensrc/thrift/DataSinks.thrift

```diff
@@ -146,6 +146,7 @@ struct TOlapTableSink {
     13: required Descriptors.TNodesInfo nodes_info
     14: optional i64 load_channel_timeout_s // the timeout of load channels in second
     15: optional bool is_lake_table
+    16: optional string txn_trace_parent
```

Comment:
Does the length of the string will too long?

Reply:
no, only fixed 55 bytes